index.html

```diff
@@ -35,6 +36,99 @@ <h2>
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+            <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+    </div>
+    <div class="aboutme">
+        <h2>About me</h2>
+        <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <div>Get my resume</div>
+    </div>
+    <hr>
+    <div class="list">
+        <ul>
+            <li><img src="/Images/Rectangle55.png" alt="rectangle"> Language</li>
```

Comment:
It seems there is a path issue in for your images 😥 Please find the bug causing this and apply its solution 🤓

![Captura](https://user-images.githubusercontent.com/15807343/148471779-d444fdfc-c828-4c6d-a4db-84691bf05910.PNG)